pkg/body-based-routing/handlers/request.go

```diff
@@ -60,6 +54,29 @@ func (s *Server) HandleRequestBody(ctx context.Context, body *eppb.HttpBody) (*e
 	}
 
 	metrics.RecordSuccessCounter()
+
+	if s.streaming {
+		return &eppb.ProcessingResponse{
+			Response: &eppb.ProcessingResponse_RequestHeaders{
+				RequestHeaders: &eppb.HeadersResponse{
+					Response: &eppb.CommonResponse{
+						ClearRouteCache: true,
+						HeaderMutation: &eppb.HeaderMutation{
+							SetHeaders: []*basepb.HeaderValueOption{
+								{
+									Header: &basepb.HeaderValue{
+										Key:      "X-Gateway-Model-Name",
```

Comment:
nit: define a constant instead of hardcoding

Reply:
Done